derive/src/lib.rs

```diff
@@ -24,10 +25,22 @@ use syn::{braced, parse_macro_input, Error as SynError, Ident, LitInt, LitStr, T
 /// ethcontract::contract!("build/contracts/MyContract.json");
 /// ```
 ///
+/// Alternatively, an etherscan URL can be specified. In this case the ABI will
+/// be retrieved and used to generate type-safe contract bindings:
+///
+/// ```ignore
+/// ethcontract::contract!("etherscan:0x0001020304050607080910111213141516171819");
+/// // or
+/// ethcontract::contract!("https://etherscan.io/address/0x0001020304050607080910111213141516171819");
```

Comment:
Would this work for rinkeby etherscan as well?

Reply:
Currently no, but we can add support for it.